overlay.py

```diff
@@ -139,13 +149,16 @@ def __init__(self, bike, width=1280, height=740):
 		configs = read_configs()
 		bike_version = bike or configs["bike"] or DEFAULT_BIKE
 		self.data = DataFactory.create(bike_version)
+		self.device = configs["device"]
 
 		self.client = mqtt.Client()
 		self.client.on_connect = self._on_connect
 		self.client.on_disconnect = self.on_disconnect
-		self.client.on_message = self._on_message
 		self.client.on_log = self.on_log
 
+		self.set_callback_for_topic_list(self.data.get_topics(), self.on_data_message)
+		self.client.message_callback_add(str(DAShboard.recording), self.on_recording_message)
```

Comment:
Why not actually include this line inside `set_callback_for_topic_list`? Also, this is really handy! Removes the need for a massive if statement inside the `on_message` function.

Reply:
Do you mean something like `self.set_callback_for_topic_list([str(DAShboard.recording)], self.on_recording_message)`?

Reply:
Yes that will work for now. Later on when more topics need their own handler, we would need to store this into a map/dictionary.